.doc_gen/metadata/bedrock-runtime_metadata.yaml

```diff
@@ -16,345 +16,287 @@ bedrock-runtime_Hello:
   services:
     bedrock-runtime: {InvokeModel}
 
-bedrock-runtime_InvokeModel:
-  title: Invoke the specified &BR; model to run an inference
-  title_abbrev: Invoke a model
-  synopsis: invoke an &BR; model to run an inference.
+bedrock-runtime_InvokeAmazonTitanImageGeneratorForImageGeneration:
+  title: Invoke the Amazon Titan Image Generator model on &BR; for image generation
+  title_abbrev: Image generation with Amazon Titan Image Generator
+  synopsis: invoke the Amazon Titan Image Generator model on &BR; for image generation.
   category:
   languages:
-    SAP ABAP:
+    Java:
       versions:
-        - sdk_version: 1
-          github: sap-abap/services/bdr
+        - sdk_version: 2
+          github: javav2/example_code/bedrock-runtime
           excerpts:
-            - description: Invoke a Claude V2 foundation model. This example uses features of /US2/CL_JSON which might not be available on some NetWeaver versions.
+            - description: Invoke asynchronously the Amazon Titan Image Generator model to generate images.
```

Comment:
Small change: these would be more consistent as "Asynchronously invoke" throughout.